packages/fast-components-react-msft/src/select-option/select-option.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Select option",
+    "description": "A select option component's schema definition.",
+    "type": "object",
+    "id": "select-option",
```

Comment:
This ID must be changed to `@microsoft/fast-components-react-msft/select-option`

Reply:
done